firmware/application/apps/ui_looking_glass_app.hpp

```diff
@@ -173,7 +173,7 @@ class GlassView : public View {
     Labels labels{
         {{0, 0 * 16}, "MIN:     MAX:     LNA   VGA  ", Color::light_grey()},
         {{0, 1 * 16}, "RANGE:       FILTER:     AMP:", Color::light_grey()},
-        {{0, 2 * 16}, "PRESET:", Color::light_grey()},
+        {{0, 2 * 16}, "P:", Color::light_grey()},
```

Comment:
Maybe use `\u007F` but P is good enough. :)

Reply:
I also tried without P: at all X-D